egs/multi_en/s5/run.sh

```diff
@@ -0,0 +1,236 @@
+#!/bin/bash
+
+# Copyright 2016  Allen Guo
+# Apache 2.0
+
+. cmd.sh
+. path.sh
+
+# corpora
+ami=/export/corpora4/ami/amicorpus
+fisher="/export/corpora3/LDC/LDC2004T19 /export/corpora3/LDC/LDC2005T19 \
+  /export/corpora3/LDC/LDC2004S13 /export/corpora3/LDC/LDC2005S13"
+librispeech=/export/a15/vpanayotov/data
+swbd=/export/corpora3/LDC/LDC97S62
+tedlium=/export/corpora5/TEDLIUM_release1
+wsj0=/export/corpora5/LDC/LDC93S6B
+wsj1=/export/corpora5/LDC/LDC94S13B
+
+# standalone evals
+eval2000="/export/corpora/LDC/LDC2002S09/hub5e_00 /export/corpora/LDC/LDC2002T43"
+rt03="/export/corpora/LDC/LDC2007S10"
+
+# general options
+stage=1
+multi=multi_a
+srilm_opts="-subset -prune-lowprobs -unk -tolower -order 3"
+notify=". notify.sh"  # notify.sh is in .gitignore
```

Comment:
Yep, forgot to mention that in the README. The idea is that you could create a `notify.sh` and immediately be able to use it in the run script. Might be feature creep though.